front/app/modules/index.ts

```diff
@@ -15,6 +14,8 @@ import machineTranslationsConfiguration from './commercial/machine_translations'
 import similarIdeaConfiguration from './commercial/similar_ideas';
 import clusteringsConfiguration from './commercial/clusterings';
 
+import projectVisibilityConfiguration from './free/project_visibility';
```

Comment:
You forgot to update line 91 (cannot comment directly the line bc it's tooo far).